frontend/src/services/graphql/ShareObject/index.js

```diff
@@ -1,11 +1,2 @@
-export * from './addSharedItem';
-export * from './approveShareObject';
 export * from './createShareObject';
-export * from './deleteShareObject';
-export * from './getShareObject';
-export * from './getShareRequestsFromMe';
 export * from './getShareRequestsToMe';
```

Comment:
Just to clarify, we need to export the `getShareRequestsToMe` to the global services because it is used in the `Datasets` module right?